.github/workflows/e2e_and_lighthouse_performance_tests.yml

```diff
@@ -63,6 +63,7 @@ jobs:
           path: /home/runner/work/oppia/build_files.zip
           retention-days: 1
   e2e_test:
+    name: E2E tests
```

Comment:
It might be better not to change this for this PR. I think it's causing the e2e tests not to run on CI.

If simpler, have the acceptance tests run as a separate CI check instead. (It's fine if they need to recompute the build etc. We can optimize later.)

Reply:
Sure no problem. I'll divide the CI runs into seperate files